test/test_forking.rb

```diff
@@ -23,6 +23,6 @@ def trigger_gc
 
 MiniRacer::Context.new.dispose
 
-Process.wait fork { puts @ctx.eval("a"); @ctx.dispose; puts Process.pid; trigger_gc; puts "done #{Process.pid}" }
+Process.wait fork { puts @ctx.eval("a"); @ctx.dispose; puts Process.pid; trigger_gc; puts "done #{Process.pid}" } if Process.respond_to?(:fork)
```

Comment:
The `if` should be on its own line for readability

Reply:
put `if` on its own line